### PR TITLE
Provision

### PR DESCRIPTION
**Pull request by cweibel**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,5 +18,11 @@ clean:
 	rm -f terraform.tfstate
 	rm -fR .terraform/
 
+prepare:
+	./provision/prepare-provision
+
+provision:
```
Comment (longnguyen11288) [inline Makefile]: i think you can make provision depend on prepare?

File: aws-cf-install.tf [hunk 2]
```diff
@@ -56,7 +56,7 @@ module "cf" {
   aws_key_path = "${var.aws_key_path}"
   aws_vpc_id = "${module.vpc.aws_vpc_id}"
   aws_internet_gateway_id = "${module.vpc.aws_internet_gateway_id}"
-  aws_route_table_public_id = "${module.vpc.aws_route_table_public_id}"
+	aws_route_table_public_id = "${module.vpc.aws_route_table_public_id}"
```
Comment (longnguyen11288) [inline aws-cf-install.tf]: wild tab?

File: test-fixtures/terraform.testplan [hunk 3]
```diff
@@ -23,7 +23,7 @@ Note: You didn't specify an "-out" parameter to save this plan, so when
     ebs_block_device.1420233019.volume_size:           "" => "40"
     ebs_block_device.1420233019.volume_type:           "" => "<computed>"
     ephemeral_block_device.#:                          "" => "<computed>"
-    instance_type:                                     "" => "m3.medium"
+    instance_type:                                     "" => "m1.medium"
```
Comment (longnguyen11288) [inline test-fixtures/terraform.testplan]: isnt it m3 now?

Comment (cweibel) [inline test-fixtures/terraform.testplan]: yup, not sure how that got lost, swear I changed that... in a different branch, ha.

File: Makefile (hunk 1 above)
Comment (cweibel) [inline Makefile]: I want to be able to edit the copy local to the server of the provision script without requiring it to match my laptop copy so I can still run make provision.  It was nice for debugging.

**Pull request merged**